### ISSUE TITLE
VS Code Windows Path Issue (duplicate drive letter in path)

### ISSUE THREAD

**Issue by aarondrabeck** [Polymer/vscode-plugin#28]
I am trying to use the new vs code plugin in Windows 10 and get the following error on all imports.  The path is known good as it is from an app in production and it runs just fine locally.   The error suggests something is adding a duplicate drive letter "**m:\m:\**".    My co-workers are also getting the same error.  Any ideas?

my-app.html
```
<link rel="import" href="../bower_components/polymer/polymer.html">
...

Error:
```
`[polymer] Unable to load import: ENOENT: no such file or directory, open 'm:\m:\00 - Sourcecode\grandmaster-nosubmodules\Websites\xxx.domain.com\project\bower_components\polymer\polymer.html`

**Comment by LostInBrittany** [comment]
It should be corrected with #24

**Comment by LostInBrittany** [comment]
Sorry, no, the 0.0.2 version and #24 don't correct it, you still need to modify `server.js` according to #16 in order to fix it. You need to modify `getWorkspacePathToFile` function:

```
function getWorkspacePathToFile(doc: {uri: string}): string|undefined {
  const match = doc.uri.match(/^file:\/\/(.*)/);
  if (!match || !match[1] || !workspaceRoot) {
    return undefined;
  }
  match[1] = match[1].replace(/^\/(\D)(\%3A\/)/,'$1:/') //convert %3A to : only if uri starts /[letter]%3A/
  return path.relative(workspaceRoot, match[1]);
}
```

And then it works (at least for me)

**Comment by LostInBrittany** [comment]
My bad... It is indeed corrected in #24, but the #24 hasn't been still put in a release.
So for the moment you need to edit the `server/server.js` file in your extension, and manually apply #24 by modifying `getWorkspacePathToFile` function: 

```
function getWorkspacePathToFile(doc: {uri: string}): string|undefined {
  // We only support file urls. Extract everything after file:///
  // Note the third slash. We pull that out too. See the absolute path
  // generation code below.
  const match = doc.uri.match(/^file:\/\/\/?(.*)/);
  if (!match || !match[1] || !workspaceRoot) {
    return undefined;
  }
  // Decode the URI encoding. TODO: something something unicode?
  const decodedPath = decodeURIComponent(match[1]);
  // We've stripped off the leading `/`, which on unix means that we have a
  // relative-looking path, but on Windows we still have an absolute path that
  // starts with e.g. `C:\`. We can normalize these two by forcing both into
  // an absolute path, like so:
  const absolutePath = path.resolve('/', decodedPath);
  // But what the editor service really wants is a path relative to the
  // workspace root. So do that.
  return path.relative(workspaceRoot, absolutePath);
}
```

**Comment by rictic** [comment]
Released as `0.1.1`!

**Comment by aarondrabeck** [comment]
Works great!  Thanks for the speedy release.
